specs/windows/nt_info.table

```diff
@@ -0,0 +1,23 @@
+table_name("nt_info") description(
+    "Display basic NT domain information of a Windows machine.")
+    schema([
+      Column("name", TEXT, "The label by which the object is known."),
+      Column("client_site_name",
+             TEXT,
+             "The name of the site where the domain controller is configured."),
+      Column("dc_site_name",
+             TEXT,
+             "The name of the site where the domain controller is located."),
+      Column("dns_forest_name", TEXT, "The name of the root of the DNS tree."),
+      Column("domain_controller_address",
+             TEXT,
+             "The IP Address of the discovered domain controller.."),
+      Column("domain_controller_name",
+             TEXT,
+             "The hostname name of the discovered domain controller."),
```

Comment:
Would it be more clear to use `domain_controller_hostname` instead?

Reply:
I'm happy to change some of these, but do we care if they are mirrors of their WMI property counterparts, or does this not matter?